vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/AlertActions.java

```diff
@@ -101,18 +100,34 @@ public boolean waitForAlert(WebDriver webDriver)
     @Override
     public void processAlert(Action action)
     {
-        Alert alert = switchToAlert(webDriverProvider.get());
-        if (alert != null)
-        {
-            action.process(alert, webDriverManager);
-        }
+        processAlertIfPresent(alert -> action.process(alert, webDriverManager));
+    }
+
+    @Override
+    public void type(String text)
+    {
+        processAlertIfPresent(alert -> alert.sendKeys(text));
```

Comment:
it makes sense to add some logging:
```suggestion
        processAlertIfPresent(alert -> {
            LOGGER.atInfo().addArgument(text).log("Typing text '{}' into the alert");
            alert.sendKeys(text);
        });
```